### DISCUSSION TITLE
Mesh network question Xplora watch

### DISCUSSION THREAD

**Discussion by smarthomefamilyverrips** [Ludy87/xplora_watch#168] [Q&A]
@Ludy87 and @Milenco can I ask if either of you uses a mesh network? I have the problem that my X6Play watches not see my ASUS aimesh network as a single network showing once the SSID but instead shows me every single node making it show the same SSID 4 times (once for each node/accesspoint) ... so was wondering if one of you experienced the same

**Comment by Milenco** [comment]
Currently not using a mesh network anymore so can't really help here @smarthomefamilyverrips .

**Reply by smarthomefamilyverrips** [reply]
Thanks for replying @Milenco 👍🏻
